virtkvmqe/migration_test_plan/migration/test_scenarios/rhel7_10052.py

```diff
@@ -50,7 +50,8 @@ def run_case(params):
 
     test.main_step_log('4. On source host, check the migration status')
     check_info = do_migration(remote_qmp=src_remote_qmp,
-                              migrate_port=incoming_port, dst_ip=dst_host_ip)
+                              migrate_port=incoming_port, dst_ip=dst_host_ip,
+                              downtime_val=downtime, speed_val=speed)
```

Comment:
NameError: global name 'downtime' is not defined
similar to rhel7_10044,  maybe it should be as following:
downtime_val='300', speed_val=speed), with this update, case pass
--> Running case(1/1): RHEL7-10052-[migration] Measure the migration speed(max-bandwidth) |('----transferred_ram_cal:', 1831.136281967163)
('----total_time_cal:', 36.401)
('----speed_cal:', 50.30455981888308)
(2 min 28 sec)--- PASS.